platforms/opencl/src/OpenCLNonbondedUtilities.cpp

```diff
@@ -356,7 +356,7 @@ void OpenCLNonbondedUtilities::prepareInteractions(int forceGroups) {
     lastCutoff = kernels.cutoffDistance;
     context.getQueue().enqueueReadBuffer(interactionCount.getDeviceBuffer(), CL_FALSE, 0, sizeof(int), pinnedCountMemory, NULL, &downloadCountEvent);
 
-    #if __APPLE__ && defined(__aarch64__)
+    #if WIN32 || __APPLE__ && defined(__aarch64__)
```

Comment:
```suggestion
    #if WIN32 || (__APPLE__ && defined(__aarch64__))
```

Currently, it looks like we're only permitting ARM64 versions of both Windows and Mac. A parenthesis would more explicitly clarify our intent.

Reply:
Operator precedence and short circuiting allows any version of `WIN32 OR (APPLE AND ARM64)`. The brackets will make it more explicit.